opta-mern-stack/client/opta-frontend-server.yaml

```diff
@@ -0,0 +1,16 @@
+name: mernfrontend
+environments:
+  - name: awsenv
+    path: "../awsenv.yaml"
+modules:
+  - name: mernfrontend
+    type: k8s-service
+    port:
+      http: 80
+    image: AUTO
+    env_vars:
+      - name: "API_ENDPOINT"
+        value: "http://opta-atlassearchenv3-lb-d60e914f75283cca.elb.us-east-1.amazonaws.com/mernbackend"
```

Comment:
We have those instructions (L82 of README); the LB URL string is for helping users understand what sort of string they need to paste here, its value is not relevant since its already torn down.

Reply:
ok I got it but I think we can we explicitly mention in the readme that `opta-frontend-server.yaml ` needs to be updated with the LB value.
and may be replace this line with something like `<replace with load_balancer_raw_dns>`

Reply:
Done